lms/static/sass/discussion/views/_thread.scss

```diff
@@ -9,197 +9,301 @@
+
+        &:hover, &:focus {
+            border-color: #222;
+        }
+    }
+}
+
+// Styling for discussion threads
+.discussion-thread {
+    padding: 0;
+    margin-bottom: $baseline;
+    @include transition(all .25s linear 0s);
+
+    p {
+        margin-bottom: 0;
+    }
+
     .thread-main-wrapper, .thread-responses-wrapper {
-      padding: $baseline;
+        padding: $baseline;
     }
-  }
 
-  .btn-brand {
-    @include blue-button;
-    display: inline-block;
-    padding-bottom: ($baseline/10);
-    height: 37px;
+    .discussion-article {
+        @include transition(all .2s linear 0s);
+        border: 1px solid $forum-color-border;
+        border-radius: 3px;
+        min-height: 0;
+        background: $forum-color-background;
+        box-shadow: 0 1px 0 $shadow;
+        @include transition(all .2s linear 0s);
+
+        .thread-wrapper {
+            @include border-radius(3px, 3px, 0, 0);
+            position: relative;
+            overflow-x: hidden;
+            overflow-y: auto;
+            max-height: 600px;
+            background-color: $forum-color-background;
+
+            .discussion-post {
+
+                .inline-comment-count {
+                    @extend %ui-depth2;
+                    position: relative;
+                    float: right;
+                    display: block;
+                    height: 27px;
+                    margin-top: 6px;
+                    margin-right: 8px;
+                    padding: 0 8px;
+                    border-radius: ($baseline/4);
+                    font-size: 12px;
+                    font-weight: 400;
+                    line-height: 25px;
+                    color: #888;
+                }
+            }
+
+            .responses {
+                header {
+                    padding-bottom: 0;
+                    margin-bottom: ($baseline*0.75);
+
+                    .posted-by {
+                        float: left;
+                        margin-right: ($baseline/4);
+                        font-size: 16px;
+                    }
+                }
+
+                .response-body {
+                    margin-bottom: 0.2em;
+                    font-size: 14px;
+                }
+            }
+
+            .discussion-reply-new {
+                .wmd-input {
+                    height: 120px;
+                }
+            }
+
+            // Content that is hidden by default in the inline view
+            .post-extended-content {
+                display: none;
+            }
+        }
+
+        .post-tools {
+            box-shadow: 0 1px 1px $shadow inset;
+            background: $gray-l6;
+
+            &:hover {
+                background: #fcfcfc;
+
+                .icon {
+                    color: $link-hover;
+                }
+            }
+
+            a {
+                display: block;
+                padding: ($baseline*0.25) $baseline;
+                font-size: 12px;
+                line-height: 30px;
```

Comment:
Not yet. I need to come up with a scheme for handling line height across both the Pattern Library and the old sass. For this PR I'm going to leave it alone.